### ISSUE TITLE
Nvidia driver 526.47 breaks vseeface

### ISSUE THREAD

**Issue by tomaae** [emilianavt/VSeeFaceReleases#21]
Hi,
I'm unable to start up vseeface anymore. It will just go past unity logo and freezes.
There is nothing useful in logs, it there a way to enable debug or get more information?

**Comment by tomaae** [comment]
Found the issue
New nvidia driver 526.47 breaks vseeface. works with 522.25

**Comment by emilianavt** [comment]
Hi, it's a known issue:
https://twitter.com/emiliana_vt/status/1585750267944992777

Hopefully it can be fixed from the driver side.

**Comment by tomaae** [comment]
Hopefully, I will test it on next release to see

**Comment by ArconusNocturn** [comment]
Issue is still present on current nVidia driver version 528.24 as well.

**Comment by TeitoShiota** [comment]
Hey, just got a 4080 and VSeeFace no longer started. I tried all the fixes on the website, but to no avail. 

In the end, for fun I turned on windows 8 compatibility mode and VSeeFace was finally able to start. 

Can't gurantee it will work for others, tried replicataing it by turning compatibility mode off and it immediately stopped working again.

GPU: Asus TUF 4080 OC 
VseeFace ver: 1.13.38b
Nvidia driver ver: 828.24
